Cue/app/common/SwitchTableRow.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import React from 'react'
+import { Platform, Switch, Text } from 'react-native'
+
+import CueColors from './CueColors'
+
+import TableRow from './TableRow'
+
+
+const styles = {
+  container: {
+    flex: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    paddingVertical: Platform.OS === 'android' ? 16 : undefined,
+  },
+  text: {
+    color: CueColors.primaryText,
+    fontSize: Platform.OS === 'android' ? 16 : 17,
+  },
+}
+
+
+export default class SwitchTableRow extends React.Component {
+  props: {
+    text: string,
+    value: boolean,
+    onPress?: (value: boolean) => void,
+  }
+
+  render() {
+    return (
+      <TableRow style={styles.container} >
+        <Text style={styles.text}>
+          {this.props.text}
+        </Text>
+        <Switch
+          onTintColor={CueColors.primaryTint}
+          tintColor={CueColors.lightGrey}
+          thumbTintColor={(Platform.OS === 'android' && this.props.value) ?
+                            CueColors.primaryTint : CueColors.veryLightGrey}
+          value={this.props.value}
+          onValueChange={this.props.onPress} />
+      </TableRow>
```

Comment:
Style nitpicks:

* Leaving `tintColor` unspecified and using the platform defaults is fine, `lightGrey` gives the switch on iOS a really dark border
* Defining `thumbTintColor` for iOS makes the default shadow on the switch go away, which makes the switch kind of blend in instead of looking like a switch:
<img width="775" alt="screen shot 2017-03-24 at 4 11 12 pm" src="https://cloud.githubusercontent.com/assets/13400887/24311904/8b848d44-10ac-11e7-9f6f-95ed0ff0bc81.png">
<img width="773" alt="screen shot 2017-03-24 at 4 13 19 pm" src="https://cloud.githubusercontent.com/assets/13400887/24311961/d4145d5a-10ac-11e7-9d9b-835750b55d8f.png">


* Use a lighter tint like `'#BED1EA'` (which is `primaryTint` at 30% opacity) for the `onTintColor` for Android to keep the switch track distinct from the grabber:
<img width="806" alt="screen shot 2017-03-24 at 4 08 38 pm" src="https://cloud.githubusercontent.com/assets/13400887/24311819/2db0809c-10ac-11e7-8a38-ae41c0d37a1f.png">